### DISCUSSION TITLE
Webhooks are not triggered on purchase action

### DISCUSSION THREAD

**Discussion by grzegorz-bak** [community/community#24704] [Apps, API and Webhooks]
<p>Hello everyone,</p>
<p>I’m new here on Support Community, but I believe I can count on your support and I wish to be able to provide support for the others in future.</p>
<p>Why I’m here? I’ve recently published an application in the GitHub marketplace, dedicated to data backup. I found around 10 days ago that webhook requests are not triggered when someone installs the application. In our provisioning system, we didn’t get the purchase request, so we don’t know about users we need to create accounts for. What’s weird in that case - all other actions (changed, canceled) properly triggers the webhook. Most of them are responded with 404 because there is no such user on our side and that’s how I get to know about the issue. I was asking GitHub support but for the last few days, I didn’t get any response. Did you ever meet a similar case? What should I do to get webhook triggered back on purchased action?</p>
<p>Thank you in advance!</p>
<p>Regards,<br>
Greg</p>

**Comment by ivailop** [comment]
<p>We discovered exactly the same problem few days ago!<br>
I also filed a ticket at <a href="https://support.github.com/" rel="noopener nofollow ugc">https://support.github.com/</a>… but have not received any response back from GitHub support yet.</p>

**Comment by saquibkhan** [comment, marked as answer]
<p>Hi,</p>
<p>I am from Engg Team, We are working on this issue and it should be fixed ASAP.</p>

**Comment by grzegorz-bak** [comment]
<p>Hello <a class="mention" href="/saquibkhan">@saquibkhan</a>,</p>
<p>thank you for your reply - it was the only information we get about the solution to that issue. I didn’t get any information from the support that the issue has been confirmed or you’re working on it. From my side, it looks GitHub has started working on the issue after I was posting here. I can confirm the issue has been fixed but we’ve lost at least several installations of the application.</p>
<p>Regards,<br>
Greg</p>

**Comment by saquibkhan** [comment]
<p><a class="mention" href="/grzegorz-bak">@grzegorz-bak</a>  we will check if we can help you here, may i know the app name?</p>

**Comment by grzegorz-bak** [comment]
<p><a class="mention" href="/saquibkhan">@saquibkhan</a> sure, the application name is <a href="http://GitProtect.io" rel="noopener nofollow ugc">GitProtect.io</a> Backup: <a href="https://github.com/marketplace/gitprotect-io" class="inline-onebox" rel="noopener nofollow ugc">GitProtect.io Backup · GitHub Marketplace · GitHub</a>.</p>

**Comment by saquibkhan** [comment]
<p><a class="mention" href="/grzegorz-bak">@grzegorz-bak</a> we have a list of users who have tried installation during the time this issue existed and might have faced the issue during installation. Do you think sharing this list will be of any help to you for reconciliation? If so then our support team will be able to share the same.</p>

**Comment by grzegorz-bak** [comment]
<p><a class="mention" href="/saquibkhan">@saquibkhan</a> excuse me for the delayed response. Meanwhile, I get the list from your support team and it will help us to reach out to those users and help them to install the application. I really appreciate your help, thank you very much!</p>
